fractions/javaee/ee/README.adoc

```diff
@@ -0,0 +1,7 @@
+# EE
+
+The EE fraction is an internal fraction used to support
+other higher-level fractions. The EE fraction does *not*
+imply the totality of Java EE support. If specific Java EE
+technologies are required, they should be individually addressed,
```

Comment:
Avoid passive tense if possible, especially because it is not always obvious who is the actor in the sentence. I do not know what _addressed_ means in this context, or who is the one doing it, but my guess is the sentence should be like this:

_If you require support for specific Java EE technologies, address them individually, for example `jaxrs`, `cdi`, `datasources`, or `ejb`._

We also always use Oxford commas in lists.